content/blog/20240430_fluentd-v1.17.0-has-been-released.md

```diff
@@ -35,14 +35,36 @@ Please see [the document](https://docs.fluentd.org/input/tail#glob_policy) and [
 
 ### `out_http`: Support AWS Signature Version 4 authentication
 
+In this release, we added a new option `aws_sigv4` for the [method](https://docs.fluentd.org/output/http#method) setting of [out_http](https://docs.fluentd.org/output/http) plugin.
+
+By using this option, `out_http` can use [AWS Signature Version 4](https://docs.aws.amazon.com/AmazonS3/latest/API/sig-v4-authenticating-requests.html).
+
+For example, this allows `out_http` to write to [Amazon OpenSearch Ingestion](https://docs.aws.amazon.com/opensearch-service/latest/developerguide/ingestion.html).
+
+Please see [the document](https://docs.fluentd.org/output/http#method) and [#4459](https://github.com/fluent/fluentd/pull/4459) for more information.
+
 ### `out_http`: Add option to reuse connections
 
+In this release, we add a new option [reuse_connections](https://docs.fluentd.org/output/http#reuse_connections) for [out_http](https://docs.fluentd.org/output/http) plugin.
+
+This option will improve performance of `out_http`.
+
+Please see [the document](https://docs.fluentd.org/output/http#reuse_connections) and [#4330](https://github.com/fluent/fluentd/pull/4330) for more information.
+
 ### `in_http`: Recognize CSP reports as JSON data
 
+In this release, we make the data of the request where the `Content-Type` is `application/csp-report` be considered JSON by default.
```

Comment:
```suggestion
In this release, we make the data type of the request where the `Content-Type` is `application/csp-report` to be considered as JSON by default.
```

The core verb should be "is" in this sentence. So, the infinitive form of "be" should be treated as to infinitive here.
In addition, the data should be written more concrete words such as "the data type".

Reply:
Thanks!
I don't understand one point about `to` infinitive.

> The core verb should be "is" in this sentence.

I intended the following as a basic sentence, so I thought it was `make`.
(I added `where` to this sentence.)

```
we make the data be considered JSON by default.
```

Is something fundamentally wrong?

Reply:
Oh, my bad.
I searched this type of sentence structure with my grammar text:
https://www.amazon.co.jp/dp/4010312971
(But you don't need to buy or read this. This is too huge to comprehend whole things :-P ) 
make O C(to-infinitive) or make O C(infinitive) would be correct.
Also, make O C(-ing: present participle) or make O C(-ed: past particible) should be correct.
You mean that Fluentd core team made the data _forcibly_ to process `application/csp-report` as JSON?
This should be correct.

Reply:
Then, my suggestion is:

```suggestion
In this release, we make the data type of the request where the `Content-Type` is `application/csp-report` be considered JSON by default.
```

Reply:
I see. Thanks!

> You mean that Fluentd core team made the data forcibly to process `application/csp-report` as JSON?

I mean that Fluentd core team made the data to process `application/csp-report` as JSON **by default**.

I'm wondering if the nuance of "forcibly" is too strong.
If so, it may be an inappropriate expression since this is about the default behavior.

Reply:
For person, the forcibly nuance should be too string.
However, in this case, we use this expression for object/functionality. 
So, it's nuance is like coercion that is casting type from one to another.

Reply:
I see! Then, it looks fine with the current suggestions.
Thanks so much!